src/main/java/mate/academy/spring/mapper/impl/request/UserRequestMapper.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.mapper.impl.request;
+
+import mate.academy.spring.mapper.DtoRequestMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserRequestMapper
+        implements DtoRequestMapper<UserRequestDto, User> {
+    @Override
+    public User fromDto(UserRequestDto dto) {
+        User user = new User();
+        user.setEmail(dto.getEmail());
+        user.setPassword(dto.getPassword());
+        user.setSalt(dto.getSalt());
```

Comment:
Setting the salt directly from UserRequestDto is not recommended. It's better to generate a new salt for each user during the registration process.